server/channels/app/channel_bookmark_test.go

```diff
@@ -98,8 +98,13 @@ func TestUpdateBookmark(t *testing.T) {
 			HasPreviewImage: true,
 		}
 
-		th.App.Srv().Store().FileInfo().Save(th.Context, file)
-		defer th.App.Srv().Store().FileInfo().PermanentDelete(th.Context, file.Id)
+		if _, appErr := th.App.Srv().Store().FileInfo().Save(th.Context, file); appErr != nil {
+			assert.Nil(t, appErr)
+		}
```

Comment:
We actually don't need the `if` checks.

Reply:
That was my mistake... sorry for that.. I was shortening the code and forget to remove that. So sorry!